flink/src/test/java/org/apache/iceberg/flink/FlinkTestBase.java

```diff
@@ -72,8 +72,21 @@ protected TableEnvironment getTableEnv() {
     return tEnv;
   }
 
+  /**
+   * if we have a sql with '%' , for example :  SELECT * FROM mytable WHERE data LIKE 'a%'. the format operation will
+   * throw an exception, so we add a sql method to execute the original sql
+   *
+   * @param query the sql
+   * @param args  the args to format sql
+   * @return the formated sql
+   */
   protected List<Object[]> sql(String query, Object... args) {
-    TableResult tableResult = getTableEnv().executeSql(String.format(query, args));
+    String sql = String.format(query, args);
+    return sql(sql);
+  }
+
+  protected List<Object[]> sql(String sql) {
+    TableResult tableResult = getTableEnv().executeSql(sql);
```

Comment:
I'd prefer to have a basic `args` check here ( rather than introducing another similar `sql(String query, Object... args)` method: 

```java
String query = args.length > 0 ? String.format(sql, args) : sql:
TableResult tableResult = getTableEnv().executeSql(query);
```

Reply:
I update it